cmd/combine/combine.go

```diff
@@ -209,6 +211,7 @@ func loadManifest(ctx context.Context, dir string, noverify bool) (*manifestpb.C
 	var (
 		possibleValKeysDir []string
 		lastCluster        *manifestpb.Cluster
+		lastMutationHash   []byte
```

Comment:
I think you can remove this variable and just use `lastCluster.LatestMutationHash`